src/main/java/org/opensearch/index/codec/customcodecs/Lucene95CustomStoredFieldsFormat.java

```diff
@@ -132,4 +132,14 @@ Lucene95CustomCodec.Mode getMode() {
     public int getCompressionLevel() {
         return compressionLevel;
     }
+
+    public Map<Lucene95CustomCodec.Mode, CompressionMode> getCompressionModes(){
```

Comment:
@sarthakaggarwal97 strong -1, the codec supports only one mode and only one compression mode - this is what consumer sees and should see. I still don't understand how we are going to expose these APIs taking into account there is only ZIP published.

Reply:
thanks @reta for your inputs. I've refactored how we are fetching the compression mode.